app/Http/Controllers/Licenses/LicensesController.php

```diff
@@ -302,7 +302,9 @@ public function getClone($licenseId = null)
     public function getExportLicensesCsv()
     {
         $this->authorize('view', License::class);
-        \Debugbar::disable();
+
+        // Disable debugbar
+        config(['debugbar.enabled' => false]);
```

Comment:
I don't think this is working like expected. I dropped this at the top of the `DashboardController@index` method and the dashboard still had debugbar show up...

At this point I think debugbar is past taking `config` into account and `Debugbar::disable()` is actually needed to disable it...

Maybe we should make a `Helper::disableDebugbar()` that checks in debugbar is install and then disables it if it is?